tests/new_unit_tests/Commands/Site/DeleteCommandTest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Pantheon\Terminus\UnitTests\Commands\Site;
+
+use Pantheon\Terminus\Commands\Site\DeleteCommand;
+use Pantheon\Terminus\UnitTests\Commands\CommandTestCase;
+
+/**
+ * Test suite for class for Pantheon\Terminus\Commands\Site\DeleteCommand
+ */
+class DeleteCommandTest extends CommandTestCase
+{
+
```

Comment:
Minor: I _think_ my builds have been failed in the past because of this extra space: how does this pass the linting or are we not running it on tests? If we aren't, let's set it up.